src/makeCmdTasks.js

```diff
@@ -27,10 +21,9 @@ module.exports = async function makeCmdTasks(
     title: linter,
```

Comment:
Hmm... I got an error here saying `title` has to be a string -- I used `title: '' + linter`.

Reply:
Good catch! I wonder why the tests haven't caught it.

Reply:
Update to handle all the cases, with a test.